flake.nix

```diff
@@ -58,6 +58,10 @@
       url = "github:gepetto/toolbox-parallel-robots";
       flake = false;
     };
+    src-unitree-sdk2-python = {
```

Comment:
I'm using flake inputs to get sources as a temporary workaround before I release some of our software, when I need to track some branches. Here I think a standard `fetchFromGitHub` would work